libdnf/module/module_item.cpp

```diff
@@ -95,25 +94,25 @@ std::string ModuleItem::get_arch() const {
 
 
 std::string ModuleItem::get_name_stream(ModulemdModuleStream * md_stream) {
-    return libdnf::utils::sformat(
+    return fmt::format(
         "{}:{}",
         libdnf::utils::string::c_to_str(modulemd_module_stream_get_module_name(md_stream)),
         libdnf::utils::string::c_to_str(modulemd_module_stream_get_stream_name(md_stream)));
 }
 
 
 std::string ModuleItem::get_name_stream_version() const {
-    return libdnf::utils::sformat(
+    return fmt::format(
         "{}:{}:{}",
         libdnf::utils::string::c_to_str(modulemd_module_stream_get_module_name(md_stream)),
```

Comment:
Only note: fmt::format accepts `char *` but it is unable to handle nullptr. I suggest that we could replace libdnf::utils::string::c_to_str by a method that will replace nullptr by a an empty C string. Then the code will not require to allocate memory for temporary std::string

Reply:
@j-mracek Good idea. We can create a new method to replace `libdnf::utils::string::c_to_str` in such cases. 
Example of new method:
```
inline constexpr const char * null_to_empty(const char * str) noexcept {
    return str ? str : "";
}
```
From my point of view, this is a code optimization that we can do additionally (in bulk) in multiple places.

Reply:
I added a TODO to improve this later.